### ISSUE TITLE
Account settings - Add option to edit account.

### ISSUE THREAD

**Issue by tmikaeld** [owncloud/ios#713]
It's insane that the address cannot be updated on an already existing account, for providers that update tens to hundred of users with a new URL - this forces every user to re-enter all of their details for no reason.

Should also add a DELETE button inside of the account settings, it's very unclear that you can swipe to delete it.

**Comment by jesmrec** [comment]
A change in the URL connection is a not very usual use case, and for that, is better to use a redirection. Changing the URL means changing the server location, so the app handle this case as a new server.
In the next release the account menu will be improved with options like the one you mentioned and a clear cache.

**Comment by tmikaeld** [comment]
That is true in most cases, but we use Owncloud in China where they block domain access frequently, so we have to use the direct IP address and we change server quite often.
